ports/portaudio/vcpkg.json

```
@@ -1,8 +1,8 @@
 {
   "name": "portaudio",
-  "version-string": "2020-02-02",
-  "port-version": 2,
+  "version-string": "2020-02-20",
```

```suggestion
  "version-date": "2020-02-20",
```
It's better to use the appropriate [version scheme](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.reference.md#version-schemes).

---

Thanks - this is my first time updating a port and appreciate the guidance.